### PR TITLE
Eng doc update

### PR DESCRIPTION
**Pull request by hzy46**

### CODE REVIEW

File: docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md [hunk 1]
```diff
@@ -151,10 +151,13 @@ sudo echo 127.0.0.1 `hostname` >> /etc/hosts
 sudo chmod 644 /etc/hosts
 ```
 
-#### Commands with `sudo` become very slow
+#### Ansible exists because `sudo` is timed out.
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md:154]: typo

File: docs/manual/cluster-admin/installation-guide.md [hunk 2]
```diff
@@ -192,20 +322,41 @@ The `user` and `password` is the SSH username and password from dev box machine
...
 
 ``` bash
 /bin/bash quick-start-kubespray.sh
 ```
 
+If there is any problem, please double check the environment requirements first. Here we provide a requirement checker to help you verify:
+
+``` bash
+/bin/bash requirement.sh
+```
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/installation-guide.md:342]: -l ${LAYOUT} -c ${CLUSTER_CONFIG} are needed for requirement.sh

**Comment by suiguoxin** [comment]
Can we put uninstallation doc for v>= 1.0 and v< 1.0 on the same page ? It will be more clear to me.

**Comment by suiguoxin** [comment]
The guide `installation-guide` is too long to read without a clear navigation bar. 
Can we divide the  `installation from scratch` part into 3 pieces : config, install k8s, start services ? And `installation FAQs and troubleshooting` should be placed under `admin-manual->Installation Guide` too.

File: docs/manual/cluster-admin/basic-management-operations.md [hunk 3]
```diff
@@ -79,24 +79,31 @@ Generally speaking, PAI services are daemon sets, deployments or stateful sets c
...
+Then, you should manually configure `kubeconfig` in the dev box container. During installation, we have put the correct `kubeconfig` file in `~/pai-deploy/kube/config` on the dev box machine. Please copy its content to `~/.kube/config` in the dev box container:
+
+```bash
+# in the dev box container
+mkdir -p ~/.kube
+# please copy the content in `~/pai-deploy/kube/config` on the dev box machine to `~/.kube/config` here
+vim ~/.kube/config
+```
+
+Go to folder `/pai`, try to retrieve your cluster id:
```
Comment (suiguoxin) [inline docs/manual/cluster-admin/basic-management-operations.md:107]: Need to remind the users of the generated `pai-deploy/cluster-configuration` folder somewhere

Comment (hzy46) [reply docs/manual/cluster-admin/basic-management-operations.md:107]: Done

**Comment by hzy46** [comment]
> The guide `installation-guide` is too long to read without a clear navigation bar.
> Can we divide the `installation from scratch` part into 3 pieces : config, install k8s, start services ? And `installation FAQs and troubleshooting` should be placed under `admin-manual->Installation Guide` too.

Done

**Comment by hzy46** [comment]
> Can we put uninstallation doc for v>= 1.0 and v< 1.0 on the same page ? It will be more clear to me.

Done

File: docs/manual/cluster-admin/installation-guide.md (hunk 2 above)
Comment (hzy46) [reply docs/manual/cluster-admin/installation-guide.md:340]: Done

File: docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md (hunk 1 above)
Comment (hzy46) [reply docs/manual/cluster-admin/installation-faqs-and-troubleshooting.md]: Done

**Review by suiguoxin — APPROVED** [review]

**Comment by suiguoxin** [comment]
Please help to update the [release wiki](https://github.com/microsoft/pai/wiki/How-to-Create-a-PR-for-New-Release)  after the merge of this PR

**Comment by hzy46** [comment]
> Please help to update the [release wiki](https://github.com/microsoft/pai/wiki/How-to-Create-a-PR-for-New-Release) after the merge of this PR

done

**Pull request merged**